package.json

```diff
@@ -164,7 +163,6 @@
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-unused-imports": "^3.0.0",
-    "husky": "^8.0.3",
```

Comment:
😍

Reply:
I believe you'd still need to remove the husky stuff locally (if you haven't already done so)?